app/controllers/applications/process_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Applications
   class ProcessController < ApplicationController
     before_action :authenticate_user!
+    before_action :evidence_check_show_redirect, only: :confirmation
```

Comment:
Considering we now don't have Wicked in place and have control over each individual action, wouldn't it be more transparent to make the redirect in the `confirmation` action? I personally think that filters, which are part of the action's flow, loading data, etc. are always quite hidden and it's difficult to understand what the action itself does.

I'm not sure, what do you think?